docs/proposals/registry/terminology.md

```diff
@@ -0,0 +1,23 @@
+# Devfile Registry Terminology
+
+The following terms may be used throughout design proposals for devfile registries
+
+## Devfile Index Image
+
+The container image containing the devfile stacks and index.json used to bootstrap the OCI registry with devfile stacks
+
+## Devfile Registry Repository 
+
+The GitHub repository that hosts the devfile stacks for consumption within an OCI registry. For example, [devfile/registry](https://github.com/devfile/registry).
+
+## Devfile Registry Support Repository 
+
+The GitHub repository that hosts tooling for OCI devfile registries. This includes the [index generator tool](https://github.com/johnmcollier/registry-support/tree/master/index/generator), and the [index container base image](https://github.com/johnmcollier/registry-support/tree/master/oci-devfile-registry-metadata)
```

Comment:
It also contains Dockerfile and the build scripts for a registry

Reply:
Updated.